forge-gui/res/cardsfolder/upcoming/urza_chief_artificer.txt

```diff
@@ -0,0 +1,12 @@
+Name:Urza, Chief Artificer
+ManaCost:3 W U B
+Types:Legendary Creature Human Artificer
+PT:4/5
+K:Affinity:Creature.Artifact+YouCtrl:artifact creatures
```

Comment:
This doesn't work yet

The Keyword needs to be updated first
https://github.com/Card-Forge/forge/blob/abf25a50f0ab8359249355ac2169ae8598c1ab93/forge-game/src/main/java/forge/game/card/CardFactoryUtil.java#L3535